xml/schema/types/transits-2-9-6.xsd

```diff
@@ -44,7 +44,8 @@
           <xs:complexType>
             <xs:sequence>
               <xs:element name="systemName" type="systemNameType" minOccurs="0" maxOccurs="1"/>
-              <xs:element name="userName" type="userNameType" minOccurs="0" maxOccurs="1"/>
+	      <xs:element name="userName" type="userNameType" minOccurs="0" maxOccurs="1"/>
+	      <xs:element name="comment" type="userNameType" minOccurs="0" maxOccurs="1"/>
```

Comment:
Update your branch and push the changes and the PR will be updated automatically.